ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -35,8 +35,27 @@
  *                     tflite-ssd (deprecated, recommend to use mobilenet-ssd)
  * option2: Location of label file
  *          This is independent from option1
- * option3: Location of box prior file (ssd) or any option1-dependent values
+ * option3: Any option1-dependent values
  *          !!This depends on option1 values!!
+ *          for mobilenet-ssd mode:
+ *		The option3 definition scheme is, in order, the following:
+ *			- box priors location file (mandatory)
+ *			- Detection threshold (optional, default set to 0.5)
+ *			- Y box scale (optional, default set to 10.0)
+ *			- X box scale (optional, default set to 10.0)
+ *			- h box scale (optional, default set to 5.0)
+ *			- w box scale (optional, default set to 5.0)
+ *			- IOU box valid threshold (optional, default set to 0.5)
+ *		The default parameters value could be set in the following ways:
+ *		option3=box-priors.txt:0.5:10.0:10.0:5.0:5.0:0.5
+ *		option3=box-priors.txt
+ *		option3=box-priors.txt::::::
+ *
+ *		It's possible to set only few values, using the default values for
+ *		those not specified through the command line.
+ *		You could specify respectively the detection and IOU thresholds to 0.65
+ *		and 0.6 with the option3 parameter as follow:
+ *		option3=box-priors.txt:0.65:::::0.6
```

Comment:
You are mostly using tabs here while other similar comments and a few of your comments don't. Please use spaces instead of tabs for the consistency.

Reply:
Done. Sorry for that. My bad.